<ui version="4.0" >
 <class>FilterEntry_BASE</class>
 <widget class="QWidget" name="FilterEntry_BASE" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_FilterEntry" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Filter #1</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>5</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QComboBox" name="comboBox_Column" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Comparators" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text" >
              <string>is</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>is not</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>is greater than</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>is less than</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>is between</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>contains</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>does not contain</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>starts with</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ends with</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>date is</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" native="1" name="entryContainer_nonDate" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEdit_EntryA" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="entryContainer_nonDate_ampersand" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&amp;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_EntryB" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="entryContainer_Date" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="3" >
            <widget class="QComboBox" name="comboBox_PredefinedFilters" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>248</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text" >
               <string>most recent</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>before</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>after</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>between</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QDateTimeEdit" name="dateTimeEdit_A" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumDate" >
              <date>
               <year>2106</year>
               <month>2</month>
               <day>7</day>
              </date>
             </property>
             <property name="minimumDate" >
              <date>
               <year>1970</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat" >
              <string>ddd dd/MM/yyyy HH:mm:ss</string>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="entryContainer_Date_ampersand" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&amp;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QDateTimeEdit" name="dateTimeEdit_B" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumDate" >
              <date>
               <year>2106</year>
               <month>2</month>
               <day>7</day>
              </date>
             </property>
             <property name="minimumDate" >
              <date>
               <year>1970</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
             <property name="currentSection" >
              <enum>QDateTimeEdit::DaySection</enum>
             </property>
             <property name="displayFormat" >
              <string>ddd dd/MM/yyyy HH:mm:ss</string>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_RemoveFilter" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>52</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>52</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Add a new filter entry</string>
            </property>
            <property name="whatsThis" >
             <string>Add a new filter entry</string>
            </property>
            <property name="text" >
             <string>-</string>
            </property>
            <property name="flat" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_AddFilter" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>52</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>52</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Add a new filter entry</string>
            </property>
            <property name="whatsThis" >
             <string>Add a new filter entry</string>
            </property>
            <property name="text" >
             <string>+</string>
            </property>
            <property name="flat" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections/>
</ui>
